### DISCUSSION TITLE
How to set different Artwork positions

### DISCUSSION THREAD

**Discussion by nufer** [punxaphil/custom-sonos-card#229] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I use a mobile Phone with Wallpannel in Landscape mode. The Sonos Card is great in Portrait, but one has to scroll in Landscape mode.

**Describe the solution you'd like**
It should be possible to remove the artwork and to place it on the right or left side of the controlls.

- showArtwork: false
- showArtwork: left
- showArtwork: right

**Comment by punxaphil** [comment]
Please provide wireframes/mockups of how you want it to look, then reopen.

**Comment by nufer** [comment]
no Artwork: (artwork: false) 

![image](https://github.com/johanfrick/custom-sonos-card/assets/6253591/d427d617-9b90-43c9-9d69-7f56400f6b5b)

artwork right: 
![image](https://github.com/johanfrick/custom-sonos-card/assets/6253591/4cf251e6-20e4-48f1-8561-9e3cba4541ec)

and artwort left is obvious. 

You can also see my use case on the "screenshots" on the top is a menu the rest can be used for the player.

**Comment by nufer** [comment]
.. and @johanfrick i'm sorry but i'm not allowed to reopen an issue if the owner closed it.

**Comment by punxaphil** [comment, marked as answer]
You'd need [card_mod](https://github.com/thomasloven/lovelace-card-mod) for that:
```
type: custom:sonos-card
heightPercentage: auto
sections:
  - player
  - volumes
  - groups
  - grouping
  - media browser
card_mod:
  style:
    sonos-player$: |
      .container {        
        grid-template-areas:
            'header artwork'
            'controls artwork' !important;
        grid-template-columns: 2fr 1fr !important;
      }
```
**Note** minimum version for this to work: https://github.com/johanfrick/custom-sonos-card/releases/tag/v6.13.1
<img width="508" alt="image" src="https://github.com/johanfrick/custom-sonos-card/assets/26582898/b375ce75-fad2-4c13-9a96-29ef138edd61">
